runner/src/server/plugins/engine/views/components/multiinputfield.html

```diff
@@ -1,7 +1,7 @@
 {% from "input/macro.njk" import govukInput %}
 {% macro MultiInputField(component) %}
     {% for item in component.model.items %}
-    <label class="govuk-label" for="event-name">
+    <label class="govuk-label" for={{item.id}}>
```

Comment:
Does this `for` correspond to the ID of the input?

Reply:
Yeah it does 
![image](https://user-images.githubusercontent.com/97108643/213512108-d9c8780d-9d46-42c0-826b-2027b9e9bcae.png)